### PR TITLE
V1.2 adds Team Packages 

### LINKED ISSUE #35: Create a Team class

### PR DESCRIPTION
**Pull request by brandonrhan**
Include class Team, TeamName and TeamList

Lineup related codes not written yet

### CODE REVIEW

File: src/main/java/seedu/address/model/team/Team.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package seedu.address.model.team;
+
+import seedu.address.model.person.Person;
+
+public class Team {
+    private final TeamName teamName;
+    private final TeamList teamList;
+
+    Team(TeamName teamName, Person[] teamMembers) {
```
Comment (DALIN-Prog) [inline src/main/java/seedu/address/model/team/Team.java]: Hi, to be consistent with teamname.java, do you want to include access modifier for the constructor and methods?

Comment (DALIN-Prog) [inline src/main/java/seedu/address/model/team/Team.java:9]: Hi, to be consistent with teamname.java, do you want to include access modifier for the constructor and methods?

File: src/main/java/seedu/address/model/team/TeamList.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package seedu.address.model.team;
+
+import java.util.ArrayList;
+
+import seedu.address.model.person.Person;
+
+public class TeamList {
+    private ArrayList<Person> list;
+
+    TeamList() {
```
Comment (DALIN-Prog) [inline src/main/java/seedu/address/model/team/TeamList.java]: Here as well

Comment (DALIN-Prog) [inline src/main/java/seedu/address/model/team/TeamList.java:10]: Here as well

**Review by DALIN-Prog — APPROVED** [review]

File: src/main/java/seedu/address/model/team/TeamList.java (hunk 2 above)
Comment (brandonrhan) [reply src/main/java/seedu/address/model/team/TeamList.java]: Hi Lin Da, I have added the access modifier as well as the java doc comments. Thanks for pointing it out!!

**Pull request merged**